app/editor/src/features/admin/reports/ReportFormScheduler.tsx

```diff
@@ -0,0 +1,35 @@
+import { useFormikContext } from 'formik';
+import React from 'react';
+import { IReportModel } from 'tno-core';
+
+import { defaultReportSchedule } from './constants/defaultReportSchedule';
+import { ReportSchedule } from './ReportSchedule';
+import * as styled from './styled';
+
+export const ReportFormScheduler: React.FC = () => {
+  const { values, setFieldValue } = useFormikContext<IReportModel>();
+
+  React.useEffect(() => {
+    // Default to two schedules if they don't already exist.
+    if (values.schedules.length !== 2) {
+      setFieldValue('schedules', [
+        defaultReportSchedule('Schedule 1', values),
+        defaultReportSchedule('Schedule 2', values),
+      ]);
+    }
+  }, [setFieldValue, values]);
+
+  return (
+    <styled.ReportFormScheduler>
+      <h2>{values.name}</h2>
+      <p>
+        Schedule the report to automatically run and send. Configure at what time and on which day
+        of the week the report should be run.
+      </p>
+      <ReportSchedule label="Schedule 1" index={0} />
```

Comment:
Currently hardcoding to two schedules.  We can however support infinite schedules if we wanted to.